src/invert/laplace/impls/serial_tri/serial_tri.cxx

```diff
@@ -83,16 +83,16 @@ const FieldPerp LaplaceSerialTri::solve(const FieldPerp &b, const FieldPerp &x0)
 
   // Setting the width of the boundary.
   // NOTE: The default is a width of 2 guard cells
-  int inbndry = 2, outbndry=2;
+  int inbndry = mesh->xstart, outbndry=mesh->xstart;
 
   // If the flags to assign that only one guard cell should be used is set
   if (global_flags & INVERT_BOTH_BNDRY_ONE) {
-    inbndry = outbndry = 1;
+    inbndry = outbndry = mesh->xstart-1;
```

Comment:
I think this one should probably be kept, as the intention of INVERT_BOTH_BNDRY_ONE is to force the number of cells to be 1 (whether this is a good idea ever is debateable). The new code would go wrong if mesh->xstart=1.